cmd/engflow_auth/main.go

```diff
@@ -73,12 +76,50 @@ func (r *appState) build(cliCtx *cli.Context) error {
 		r.browserOpener = &browser.StderrPrint{}
 	}
 	if r.tokenStore == nil {
-		tokenStore, err := oauthtoken.NewKeyring()
+		keyring, err := oauthtoken.NewKeyring()
 		if err != nil {
-			return autherr.CodedErrorf(autherr.CodeTokenStoreFailure, "failed to open token store: %w", err)
+			return autherr.CodedErrorf(autherr.CodeTokenStoreFailure, "failed to open keyring-based token store: %w", err)
 		}
 
-		r.tokenStore = oauthtoken.NewCacheAlert(tokenStore, cliCtx.App.ErrWriter)
+		userConfigDir, err := os.UserConfigDir()
+		if err != nil {
+			return autherr.CodedErrorf(autherr.CodeTokenStoreFailure, "failed to discover user's config dir: %w", err)
+		}
+		tokensDir := filepath.Join(userConfigDir, "engflow_auth", "tokens")
+		fileStore, err := oauthtoken.NewFileTokenStore(tokensDir)
+		if err != nil {
+			return autherr.CodedErrorf(autherr.CodeTokenStoreFailure, "failed to open file-based token store: %w", err)
+		}
+
+		errorStore := &oauthtoken.FakeTokenStore{
+			StoreErr: fmt.Errorf("subcommand attempted illegal write to token storage"),
```

Comment:
General advice: don't use the word "illegal" in error messages. It freaks people out, especially newer developers. "Invalid" is often better.

Reply:
Done!